packages/playground/remote/src/lib/worker-thread.ts

```diff
@@ -186,6 +187,86 @@ export class PlaygroundWorkerEndpoint extends PHPWorker {
 	async replayFSJournal(events: FilesystemOperation[]) {
 		return replayFSJournal(this.__internal_getPHP()!, events);
 	}
+
+	async backfillStaticFilesRemovedFromMinifiedBuild() {
+		await backfillStaticFilesRemovedFromMinifiedBuild(
+			this.__internal_getPHP()!
+		);
+	}
+}
+
+/**
+ * Downloads static assets that are removed from minified WordPress builds.
+ *
+ * ## Background
+ *
+ * To load Playground faster, we ship minified WordPress builds without most CSS files, JS files, and other static assets.
```

Comment:
Let's clarify that minified builds are not mandatory but only used by default:

```suggestion
 * To load Playground faster, we default to minified WordPress builds shipped without most CSS files, JS files, and other static assets.
```